packages/angular/projects/clr-angular/src/data/datagrid/datagrid-row.ts

```diff
@@ -264,4 +266,20 @@ export class ClrDatagridRow<T = any> implements AfterContentInit, AfterViewInit
   public get _view() {
     return this.wrappedInjector.get(WrappedRow, this.vcr).rowView;
   }
+
+  @Input('ariaTextBeginningOfExpandableContent') ariaTextBeginningOfExpandableContent: string;
+  public get beginningOfExpandableContent() {
+    return (
+      this.ariaTextBeginningOfExpandableContent ||
+      `${this.commonStrings.keys.dategridExpandableBeginningOf} ${this.commonStrings.keys.dategridExpandableRowContent}`
+    );
+  }
+
+  @Input('ariaTextEndOfExpandableContent') ariaTextEndOfExpandableContent: string;
+  public get endOfExpandableContent() {
+    return (
+      this.ariaTextEndOfExpandableContent ||
+      `${this.commonStrings.keys.dategridExpandableEndOf} ${this.commonStrings.keys.dategridExpandableRowContent}`
+    );
+  }
```

Comment:
I still don't understand why these input properties are being added to `clr-dg-row` instead of `clr-dg-row-detail`?